config/crds/hive_v1_hiveconfig.yaml

```diff
@@ -118,7 +118,7 @@ spec:
               type: boolean
             managedDomains:
               description: 'ManagedDomains is the list of DNS domains that are managed
-                by the Hive cluster When specifying ''managedDNS: true'' in a ClusterDeployment,
+                by the Hive cluster When specifying ''manageDNS: true'' in a ClusterDeployment,
```

Comment:
https://github.com/openshift/hive/blob/57ac2a74054f4633ef151e43db7cd4302e649ec5/pkg/apis/hive/v1/clusterdeployment_types.go#L88

It looks like the property is actually "manageDNS"